aws/elasticache/outputs.tf

```diff
@@ -1,5 +1,5 @@
 output "sg_for_access_by_sgs_id" {
-  value = "${aws_security_group.sg_for_access_by_sgs.id}"
+  value = "${element(concat(aws_security_group.sg_for_access_by_sgs.*.id, list("")), 0)}"
```

Comment:
I dont understand this line

Reply:
This allows for an empty output otherwise terraform complains